src/test/scala/mesosphere/marathon/MarathonSchedulerActorTest.scala

```diff
@@ -187,13 +187,13 @@ class MarathonSchedulerActorTest extends AkkaUnitTest with ImplicitSender with G
 
     "ScaleApp" in withFixture() { f =>
       import f._
-      val app = AppDefinition(id = "test-app".toPath, instances = 1, cmd = Some("sleep"))
+      val app = AppDefinition(id = "test-app-scale".toPath, instances = 1, cmd = Some("sleep"))
```

Comment:
I'm not sure this is a valid path. It should start with `/` otherwise the parent is null.